### PR TITLE
Fail app launch command if the app was already running

### PR DESCRIPTION
**Pull request by berikv**

### CODE REVIEW

**Review by artem888 — APPROVED** [review]

**Review by dmitry-zaitsev — CHANGES_REQUESTED** [review]
Comment

File: maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt [hunk 1]
```diff
@@ -286,7 +286,6 @@ class IdbIOSDevice(
                 launchRequest {
                     start = idb.LaunchRequestKt.start {
                         bundleId = id
-                        foregroundIfRunning = true
```
[the remark this answers is not in the document]
Comment (berikv) [reply maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt:289]: What is continuous mode?

[the remark this answers is not in the document]
Comment (dmitry-zaitsev) [reply maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt:289]: https://maestro.mobile.dev/guides/using-maestro-cli#run-test-continuously

TL;DR: we will monitor yaml file changes and re-run test whenever there is an update

[the remark this answers is not in the document]
Comment (berikv) [reply maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt:289]: I'll test this mode

[the remark this answers is not in the document]
Comment (berikv) [reply maestro-ios/src/main/java/ios/idb/IdbIOSDevice.kt:289]: Works in continuous mode ✅

**Review by dmitry-zaitsev — APPROVED** [review]
